### DISCUSSION TITLE
My follows amount on the message tab is going up and down (70-69)

### DISCUSSION THREAD

**Discussion by joshuangdev** [ScratchAddons/ScratchAddons#3118] [Discussion]
But it's not true, it's the same every time it (Scratch Addons change the num)

**Comment by mxmou** [comment]
This is because the unread message count and the messages themselves are loaded separately, so they can be out of sync, which causes some read messages to appear on the Messaging tab. I'm not sure if there is a way to prevent this.

**Reply by joshuangdev** [reply]
Never mind its fine now
